vendor/modules.txt

```diff
@@ -69,7 +69,7 @@ github.com/prometheus/procfs/internal/util
 ## explicit
 github.com/redhat-cne/rest-api
 github.com/redhat-cne/rest-api/pkg/localmetrics
-# github.com/redhat-cne/sdk-go v0.0.0-20210630032545-4072da5111ba
+# github.com/redhat-cne/sdk-go v0.0.0-local => ../sdk-go
```

Comment:
need updated

Reply:
right, will update mod file once the sdk-go redfish-schema change is delivered.